app/src/main/assets/indoorMapControl.js

```diff
@@ -1,72 +1,119 @@
 var map;
 
 var floormapGroup;
-var markerGroup;
+var roomMarkerGroup;
+var pathMarkerGroup;
 var pathGroup;
 
 function initmap() {
-    map = new L.Map('map', {
-        crs: L.CRS.Simple,
-        minZoom: -1,
-        maxZoom: 1,
-        attributionControl: false, //Remove Attribution on bottom right
-        zoomControl: false //Remove "+" and "-" button on top left
-    });
-    floormapGroup = L.layerGroup().addTo(map);
-
-    //Listener Registration
-    map.on('click', function(ev) {
-        console.log(JSON.stringify(ev.latlng));
-    });
+	map = new L.Map('map', {
+		crs: L.CRS.Simple,
+		minZoom: -2,
+		maxZoom: 1,
+		attributionControl: false, //Remove Attribution on bottom right
+		zoomControl: false //Remove "+" and "-" button on top left
+	});
+	//Image Groups
+	floormapGroup = L.layerGroup().addTo(map);
+
+	//Path Groups
+	pathGroup = L.layerGroup().addTo(map);
+
+	//Marker Groups
+	roomMarkerGroup = L.layerGroup().addTo(map);
+	pathMarkerGroup = L.layerGroup().addTo(map);
+
+	//Listener Registration
```

Comment:
It's commented out when I was debugging, but its supposed to be there as it prints the coordinates wherever you click, so its needed when annotating classes